server/src/main/kotlin/org/itmodreamteam/myrest/server/view/assembler/ModelViewAssembler.kt

```diff
@@ -0,0 +1,7 @@
+package org.itmodreamteam.myrest.server.view.assembler
+
+interface ModelViewAssembler<in M, out V> {
+    fun toView(model: M): V
+
+    fun toListView(models: List<M>): List<V> = models.map { toView(it) }
```

Comment:
Maybe `toViewList`? Same as `toUserList`, `toObjectList`, list of views, not view of list

Reply:
Agree